### ISSUE TITLE
custom bar count

### ISSUE THREAD

**Issue by Dalkson** [marcopixel/Monstercat-Visualizer#246]
A way to set a custom bar count would be nice, i want the bars to be small but then it seems to small for the whole screen.

**Comment by marcopixel** [comment]
You can set a custom amount of bars under Settings -> Visualizer and you can even adjust the bar size and gap.

**Comment by Dalkson** [comment]
yeah but i cant get it higher than 100 is my issue, id like to keep the gap and size small but it just doesnt fill up the screen enough.

**Comment by marcopixel** [comment]
I'll have to take a look on that and see if i can do some stuff with lua to make this happen. Keep in mind tho that this will make it much harder on the CPU and i will probably lock this to be only avaliable via the **variables.ini** file if i add this as an feature.

**Comment by Dalkson** [comment]
Alright, thanks for the quick reply. How much do you think it would effect the cpu to have let’s say 200 instead of 100

**Comment by GordnCZ** [comment]
Just edit MaxValue option in visualizer.ini file under ; --- BarCount --- ; line ;-)

**Comment by gmanc2** [comment]
Is there any update to this it only goes on about half my monitor at 2x scale please extend this feature! @marcopixel
